src/api/lib/workers/accept_requests.rb

```diff
@@ -4,7 +4,7 @@ def initialize
 
   def perform
     User.current = User.find_by_login('Admin')
-    BsRequest.find_requests_to_accept.each do |r|
+    BsRequest.to_accept.each do |r|
```

Comment:
That was done in August 2013 by 7a92356c95bc7e91a105bde0017900c2c522f00a

How did we manage to not run into this earlier?^^